storage/persistent_storage/persistent-storage-ocs.adoc

```diff
@@ -7,7 +7,7 @@ toc::[]
 
 Red Hat OpenShift Container Storage is a provider of agnostic persistent storage for {product-title} supporting file, block, and object storage, either in-house or in hybrid clouds. As a Red Hat storage solution, Red Hat OpenShift Container Storage is completely integrated with {product-title} for deployment, management, and monitoring.
 
-Red Hat OpenShift Container Storage provides its own documentation library. The complete set of Red Hat OpenShift Container Storage documentation identified below is available at https://access.redhat.com/documentation/en-us/red_hat_openshift_container_storage/4.6/
+Red Hat OpenShift Container Storage provides its own documentation library. The complete set of Red Hat OpenShift Container Storage documentation identified below is available at https://access.redhat.com/documentation/en-us/red_hat_openshift_container_storage/4.7/
```

Comment:
We do have attributes for OCS in `modules/common-attributes.adoc` that I added a while back:

- :rh-storage-first: Red Hat OpenShift Container Storage
- :rh-storage: OpenShift Container Storage

Due to product names sometimes changing, it would be helpful to replace product name occurrences with the attribute. I realize that is out of scope for this PR, however.